seed/challenges/02-data-visualization-certification/react-projects.json

```diff
@@ -202,6 +265,25 @@
       "tests": [],
       "type": "zipline",
       "challengeType": 3,
+      "nameRu": "Создайте Рогулайк Данжеон-Кроулер",
```

Comment:
I will do whatever you decide at this point.
The argument for using `Рогулайк` in the task name is the fact that Dungeon Crawlers are not necessarily roguelike, and the difference is quite essential.
A random Reddit user knows that Dungeon Crawlers can be not roguelike: https://www.reddit.com/r/AndroidGaming/comments/2xvmy9/any_nonroguelike_dungeon_crawlers/
Also, you may want to check these links:
- http://www.roguebasin.com/index.php?title=What_a_roguelike_is
- http://www.roguetemple.com/roguelike-definition/

In conclusion, Make a Dungeon Crawler may seem clear for most people, but Make a Roguelike Dungeon Crawler narrows down the task, because one can search for particular term and read what should be done instead of just looking at the example and trying to figure out how it should work.

All other corrections are already made or will be made soon, so if you still insist on `Создайте игру Подземелье`, I'll use this phrase. Confirm it.